tutorial/connect-from-aks/ConnectFromAKS/README.md

```diff
@@ -0,0 +1,6 @@
+# Connect to Azure Cache for Redis from an application running in AKS
+
+## Overview
+
+This project shows how to connect to Azure Cache for Redis from an application running in AKS using a managed identity (AKS workload identity) or an access key.
```

Comment:
again be careful about saying 'managed identity'. You could just say 'a Microsoft Entra identity'